.github/workflows/integrate.yaml

```
@@ -54,6 +54,22 @@ jobs:
     - name: Run unit tests
       run: tox -vve ${{ matrix.charm }}-unit
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    strategy:
+      matrix:
+        charm:
+          - tensorboard-controller
+          - tensorboards-web-app
+    with:
+      charm-path: ./charms/${{ matrix.charm }}
```

Let's add the proper channel here so we don't forget when we actually apply this action
```suggestion
      charm-path: ./charms/${{ matrix.charm }}
      channel: "1.9/stable"
```

---

is this necessary? the default channel of the terraform module in this track is now `1.9/stable` so the action will apply from that channel. I think we can avoid setting this in 2 places.

---

From my understanding, [this line](https://github.com/canonical/charmed-kubeflow-workflows/blob/e6d94ca3fee0fc39d85dad66cfeaf19ec5ef9b53/.github/workflows/terraform-apply.yaml#L49) from the action always sets the channel from the workflow's input, and if the input doesn't exist, the default value (so "latest/stable" found [here](https://github.com/canonical/charmed-kubeflow-workflows/blob/e6d94ca3fee0fc39d85dad66cfeaf19ec5ef9b53/.github/workflows/terraform-apply.yaml#L18)) is used. Is that correct?

---

yes that makes sense^ I got in touch with the team and we agreed to have the channel set in the workflow for `track/<version>` branches, this applies to all backport PRs
good catch!

---

as for the default `channel` in the `variables.tf`, this is subject to change based on feedback from the terraform WG

---

the conversation has been resolved, we should use <version>/stable